### ISSUE TITLE
On the Pixelbook go it only works with Ubunut 20.04 LTS

### ISSUE THREAD

**Issue by mateowoetam** [yusefnapora/pixelbook-linux#72]
On the Pixelbook go, it only works with Ubuntu 20.04 LTS, I tried first on ubuntu 22.10, and the Packages wouldn't download; I tried on ubuntu 19.04, and Git wouldn't download, now I installed ubuntu 20.4, and it seems to have some issues, but it did download most stuff, It had one fail, 39 OKs, 27 changed, three skipped, and three ignored.

I would post the report but it's too long.

**Comment by TenzinJamyangZHS** [comment]
did you try Ubuntu 22.04？

**Comment by mateowoetam** [comment]
Yep, same results if I remember correctly.

**Comment by TenzinJamyangZHS** [comment]
> Yep, same results if I remember correctly.

does all work will on Ubuntu20.04？like keyboard backlight, trackpad, camera？

**Comment by mateowoetam** [comment]
I don't know, cuz I didn't get it to work and abandoned the project, my Pixelbook is back to runing ChromeOS and stored in a drawer.

**Comment by gfhdhytghd** [comment]
try delete the kernal part from the playbook.yml

**Comment by mateowoetam** [comment]
little bit late, to the comments my friend, but I resolved the issue in a different manner, I made my own repo of scrips I know work if you'd like to see: https://github.com/mateowoetam/tuxelbookgoscript

**Comment by TenzinJamyangZHS** [comment]
> little bit late, to the comments my friend, but I resolved the issue in a different manner, I made my own repo of scrips I know work if you'd like to see: https://github.com/mateowoetam/tuxelbookgoscript

audio works！thank you very much！

**Comment by TenzinJamyangZHS** [comment]
> little bit late, to the comments my friend, but I resolved the issue in a different manner, I made my own repo of scrips I know work if you'd like to see: https://github.com/mateowoetam/tuxelbookgoscript

thank you for you work, how can i undo keyboard layout change？i only want the audio work.

**Comment by mateowoetam** [comment]
yes, the script should prompt you if you want to skip that part, and you just press 's' to do so, also, even if it does install it it doesn't activate it, it simply places a new keyboard rules and layout into the configuration files so that if you want to edit your layout 'Halmak' is now available. and as for the buttons reassignment (search and assistant) you can also just skip that.

**Comment by gfhdhytghd** [comment]
I try to downgrade to kernal linux-zen 5.19 and follow the instruction on gentoo wiki , and then everything work,I'm using archlinux

**Comment by gfhdhytghd** [comment]
> > little bit late, to the comments my friend, but I resolved the issue in a different manner, I made my own repo of scrips I know work if you'd like to see: https://github.com/mateowoetam/tuxelbookgoscript
> 
> thank you for you work, how can i undo keyboard layout change？i only want the audio work.

search "pixelbook grntoo" and go to that link, scorll down, see what happens with the udev and keyboard re-mapping,  remove the udev file and refresh it
